### DISCUSSION TITLE
What is one flash at second please ?

### DISCUSSION THREAD

**Discussion by denisjoshua** [marcone/teslausb#850] [General]
Hi there, I so a list of nr. of LED flashes, but there is only 2,3,4 or 5 
What mean only one flash please ? 
Sometime I have only one flash and my Tesla can't see the USB drive 
Thanks in advance 
Denis

**Comment by WotAGitUR** [comment]
Check the first paragraph here, it describes the different blink patterns and what they mean
https://github.com/marcone/teslausb/wiki/Troubleshooting

**Comment by denisjoshua** [comment]
"the single blink flashing mode indicates it is not able to reach the archive server"
Ok thanks a lot. 
Denis
